ontotext-yasgui-web-component/src/components/ontotext-yasgui-web-component/ontotext-yasgui-web-component.tsx

```diff
@@ -628,7 +634,24 @@ export class OntotextYasguiWebComponent {
     });
   }
 
-  private destroy() {
+  private initYasguiEventHandlers(yasqe: any): void {
+    yasqe.off('query', this.onQueryBound);
+    yasqe.on('query', this.onQueryBound);
+    yasqe.off('queryResponse', this.onQueryResponseBound);
+    yasqe.on('queryResponse', this.onQueryResponseBound);
+  }
+
+  private afterInit(): void {
+    // Configure the web component
+    this.ontotextYasguiService.postConstruct(this.hostElement, this.ontotextYasgui.getConfig());
+    // Register any needed event handlers
+    // @ts-ignore
+    this.ontotextYasgui.getInstance().on('yasqeInitialized', (tab: any, yasqe: any) => {
```

Comment:
IMO is better if we not use // @ts-ignore if it is possible of course. You can remove it and rename "tab" to "_tab" in this case.

Reply:
Yes. I always forget about this possibility. I'll change it.